pkg/infrastructure/gcp/clusterapi/clusterapi.go

```diff
@@ -31,6 +30,7 @@ var _ clusterapi.PreProvider = (*Provider)(nil)
 var _ clusterapi.IgnitionProvider = (*Provider)(nil)
 var _ clusterapi.InfraReadyProvider = (*Provider)(nil)
 var _ clusterapi.PostProvider = (*Provider)(nil)
+var _ clusterapi.BootstrapDestroyer = (*Provider)(nil)
```

Comment:
In aws, I do not see this BootstapDestroyer declared https://github.com/openshift/installer/blob/master/pkg/infrastructure/aws/clusterapi/aws.go#L28-L31. Won't we need that too. /cc @r4f4

Reply:
Already done: https://github.com/openshift/installer/pull/8543